docs/content/voice-and-tone.mdx

```diff
@@ -27,13 +27,24 @@ do).
 
 ### Punctuation
 
+#### Periods
+
```

Comment:
@agglasaur we already had punctuation in this doc so I added ampersand guidance here - alternatively, could be worth moving all punctuation to formatting?

Reply:
Yeah probably makes sense in formatting, that would be my first place to look